HTTPServerProject.Tests/HTTPServerProxyTests/TestFirstID.cs

```diff
@@ -0,0 +1,103 @@
+using HTTPServerRead.Streams;
+using HTTPServerRead.Header;
+using HTTPServerRead.Body;
+using HTTPServerWrite.Request;
+using HTTPServerWrite.Streams;
+using HTTPServerProxy.FirstID;
+
+namespace HTTPServerProxyTests.FirstID;
+
+public class TestGetFirstID : IGetFirstID
```

Comment:
I think I left a comment about this on the other PR - so this might be redundant - but this class looks like a duplicate of a real class that belongs in the implementation. If it's a mock, it could it be made a lot simpler.

Reply:
Yes it is a mock - I am mocking a method that makes a GET request. I could take this method out and give the class less responsibility.